TrackerMC/inc/StrawPosition.hh

```diff
@@ -0,0 +1,9 @@
+#ifndef TrackerMC_StrawPosition_hh
+#define TrackerMC_StrawPosition_hh
+#include "Math/Vector3D.h"
+namespace mu2e {
+  namespace TrackerMC {
+    typedef ROOT::Math::Cylindrical3D<Float_t> StrawPosition; // position WRT the straw.
```

Comment:
Is ROOT::Math::Cylindrical3D<T> a TObject?  If so we should consider an alternative that is not a TObject.